module/Application/view/error/404.phtml

```diff
@@ -31,14 +31,14 @@ switch ($this->reason) {
 <?php if (isset($this->controller) && $this->controller): ?>
 
 <dl>
-    <dt><?php echo $this->translate('Controller') ?>:</dt>
+    <dt>Controller:</dt>
     <dd><?php echo $this->escapeHtml($this->controller) ?>
 <?php
 if (isset($this->controller_class)
     && $this->controller_class
     && $this->controller_class != $this->controller
 ) {
-    echo '(' . sprintf($this->translate('resolves to %s'), $this->escapeHtml($this->controller_class)) . ')';
+    echo '(' . sprintf('resolves to %s', $this->escapeHtml($this->controller_class)) . ')';
```

Comment:
For the skeleton, yes; see the issue summary. :wink:

(Plan is to potentially ship multiple skeletons, or use an installer to provide different features.)